fxprof-processed-profile/src/markers.rs

```diff
@@ -90,6 +90,21 @@ pub trait Marker {
     /// If you do see unexpected calls to this method, make sure you're not registering
     /// multiple different schemas with the same [`RuntimeSchemaMarkerSchema::type_name`].
     fn number_field_value(&self, field_index: u32) -> f64;
+
+    /// Called for any fields defined in the schema whose [`format`](RuntimeSchemaMarkerField::format) is
+    /// of [kind](MarkerFieldFormat::kind) [`MarkerFieldFormatKind::Flow`].
+    ///
+    /// `field_index` is an index into the schema's [`fields`](RuntimeSchemaMarkerSchema::fields).
+    ///
+    /// Flow identifiers are u64 values that are unique across processes.
+    ///
+    /// You can panic for any unexpected field indexes, for example
+    /// using `unreachable!()`. You can even panic unconditionally if this
+    /// marker type doesn't have any flow fields.
+    ///
+    /// If you do see unexpected calls to this method, make sure you're not registering
+    /// multiple different schemas with the same [`RuntimeSchemaMarkerSchema::type_name`].
+    fn flow_field_value(&self, field_index: u32) -> u64;
```

Comment:
Should we use a `FlowId(u64)` newtype here? Not sure if that would help any. But it might provide a surface for more doc comments